README.md

```diff
@@ -33,11 +62,11 @@ To do this, set the target (replace `<TARGET_NAME>` with your target name) value
         ...
 ```
 
-Note - `restrict_size` pads the build image to the requested size.
+<span class="tips">**Note:** `restrict_size` pads the build image to the requested size.</span>
 
-## Now compile
+## Building
 
-Invoke `mbed compile`, and specify the name of your platform and your favorite toolchain (`GCC_ARM`, `ARM`, `IAR`). For example, for the ARM Compiler 5:
+Invoke `mbed compile`, and specify the name of your target and your favorite toolchain (`GCC_ARM`, `ARM`, `IAR`). For example, for the ARM Compiler 5:
```

Comment:
I don't think we should specify the ARM compiler version # here, as this will evolve over time.